tests/integration/test_record.py

```diff
@@ -139,3 +180,8 @@ def test_record_can_be_deleted(app, record_not_yet_deleted):
         db.session.commit()
 
         assert client.get('/api/literature/333').status_code == 410
+
+
+def test_record_can_be_immediately_es_updated(app, record_not_es_refreshed):
+    with app.test_client() as client:
+        assert client.get('/api/literature/444').status_code == 200
```

Comment:
I really appreciate that you thought of documenting this with a test 👍 

On the other hand, I think it's better if we reuse the previous fixture and remove the index refresh, rather than defining a new "non-refreshed" fixture.